src/Symfony/Component/Config/Definition/Builder/NodeBuilder.php

```diff
@@ -143,8 +143,8 @@ public function end()
     /**
      * Creates a child node.
      *
-     * @param string|null $name The name of the node
-     * @param string      $type The type of the node
+     * @param string $name The name of the node
```

Comment:
Shouldn't this be reverted?

Reply:
The `NodeDefinition` class constructor allows `null`.

Reply:
yes mea culpa will revert